packages/delivery-xml-http-request/delivery.js

```diff
@@ -8,12 +9,17 @@ module.exports = (client, win = window) => ({
       req.onreadystatechange = function () {
         if (req.readyState === win.XMLHttpRequest.DONE) cb(null)
       }
+
+      const json = payload.event(event, client._config.redactedKeys)
+
       req.open('POST', url)
       req.setRequestHeader('Content-Type', 'application/json')
       req.setRequestHeader('Bugsnag-Api-Key', event.apiKey || client._config.apiKey)
+      req.setRequestHeader('Bugsnag-Integrity', 'simple ' + bytesize(json))
```

Comment:
minor nit – backticks are prefferd syntax for string interpolation (but it definitely gets compiled down to what you wrote)

```suggestion
      req.setRequestHeader('Bugsnag-Integrity', `simple ${bytesize(json)}`)
```